ReactWindows/ReactNative/Modules/Clipboard/IClipboardInstance.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Windows.ApplicationModel.DataTransfer;
+
+namespace ReactNative.Modules.Clipboard
+{
+    /// <summary>
+    /// Gets and sets information from the clipboard object.
+    /// </summary>
+    public interface IClipboardInstance
+    {
+        /// <summary>
+        /// Removes all data from the Clipboard.
+        /// </summary>
+        void Clear();
+
+        /// <summary>
+        /// Gets the current content that is stored in the clipboard object.
+        /// </summary>
+        /// <returns>Contains the content of the Clipboard.</returns>
+        DataPackageView GetContent();
```

Comment:
@rozele actually, it's not that hard because you can create the `DataPackage` which will give you your view:
```cs
var package = new DataTransfer.DataPackage();
package.SetData(DataTransfer.StandardDataFormats.Text, text);
var view = package.GetView();
```